blog/2025/keyconf-25-japan-call-for-papers.adoc

```diff
@@ -0,0 +1,23 @@
+:title: Submit to KeyConf25 Japan Call-for-Papers!
+:date: 2025-03-17
+:publish: true
+:author: Alexander Schwartz
+:preview: keyconf25-japan-cfp.png
+:summary: Submit your talks to the first-ever KeyConf in Japan! It will happend shortly before KubeCon Japan, so you can join both.
+
+The call for papers for KeyConf25 Japan is now open! link:https://sessionize.com/keyconf-2025-japan/[Submit your talks] to the first-ever KeyConf in Japan.
```

Comment:
```suggestion
The call for papers for [KeyConf 2025 Japan](https://community.cncf.io/e/mvnu59/) is now open! link:https://sessionize.com/keyconf-2025-japan/[Submit your talks] to the first-ever KeyConf in Japan.
```

Reply:
Thank you - yes, this link should be there to provide more information about the event.